Game settings error - Android, version 4.0.0

java.lang.NoClassDefFoundError: 
  at com.unciv.models.metadata.GameSettings$Companion.getSettingsForPlatformLaunchers (GameSettings.kt:122)
  at com.unciv.app.AndroidLauncher.onCreate (AndroidLauncher.kt:39)
  at android.app.Activity.performCreate (Activity.java:6666)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1118)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2732)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2844)
  at android.app.ActivityThread.-wrap12 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1572)
  at android.os.Handler.dispatchMessage (Handler.java:110)
  at android.os.Looper.loop (Looper.java:203)
  at android.app.ActivityThread.main (ActivityThread.java:6368)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1063)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:924)
Caused by: java.lang.ClassNotFoundException: 
  at dalvik.system.BaseDexClassLoader.findClass (BaseDexClassLoader.java:56)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:380)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:312)

--- a/core/src/com/unciv/models/metadata/GameSettings.kt
+++ b/core/src/com/unciv/models/metadata/GameSettings.kt
@@ -5,13 +5,12 @@ import com.badlogic.gdx.Gdx
 import com.badlogic.gdx.files.FileHandle
 import com.unciv.JsonParser
 import com.unciv.Constants
-import com.unciv.UncivGame
 import com.unciv.logic.GameSaver
 import com.unciv.ui.utils.Fonts
+import java.io.File
 import java.text.Collator
 import java.util.*
 import kotlin.collections.HashSet
-import kotlin.io.path.Path
 
 data class WindowState (val width: Int = 900, val height: Int = 600)
 
@@ -119,7 +118,7 @@ class GameSettings {
         fun getSettingsForPlatformLaunchers(base: String = ""): GameSettings {
             // FileHandle is Gdx, but the class and JsonParser are not dependent on app initialization
             // If fact, at this point Gdx.app or Gdx.files are null but this still works.
-            val file = FileHandle(Path(base, GameSaver.settingsFileName).toString())
+            val file = FileHandle(base + File.separator + GameSaver.settingsFileName)
             return if (file.exists())
                 JsonParser().getFromJson(
                     GameSettings::class.java,

